.github/workflows/e2e.yaml

```diff
@@ -36,13 +36,15 @@ on:
           - "1.27"
           - "1.28"
         default: "1.28"
-      enable_metrics:
-        type: boolean
-        default: false
+      cluster_name:
```

Comment:
Consider adding a description to this argument to the workflow_call. Something that states that if this value is an empty string or isn't specified, that implies that we are going to create a cluster. Else, if it is specified, we are going to use the existing cluster